kyo-bench/src/main/scala/kyo/bench/TestHttpServer.scala

```diff
@@ -65,6 +105,9 @@ object TestHttpServer:
                 .setMaxChunkSize(8192)
                 .setIdleTimeout(0)
                 .setTcpKeepAlive(true)
+                .setInitialSettings(
+                    new Http2Settings().setMaxConcurrentStreams(1000)
```

Comment:
I still saw some cases where Kyo would reach the max number of streams so I'm increasing the default here